src/main/java/org/ow2/proactive/procci/rest/ComputeRest.java

```diff
@@ -71,13 +74,23 @@
 
     private final Logger logger = LogManager.getRootLogger();
 
+    @Autowired
+    private CloudAutomationInstances cloudAutomationInstances;
```

Comment:
Are those services? You can think about names which contain the type of the class like InstancesRetreivalService. I am not sure if cloud automation is expressive , except if there are (currently) other InstancesRetreivalService implementations.